src/odb/src/db/dbPolygon.cpp

```diff
@@ -251,14 +251,6 @@ Polygon _dbPolygon::checkPolygon(std::vector<Point> polygon)
     return {};
   }
 
-  if (polygon[0] == polygon[polygon.size() - 1]) {
-    polygon.pop_back();
-  }
```

Comment:
I don't know if the decomposer assumes this has been done.

Reply:
good point. I added it back.